examples/vaecf_example.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018 The Cornac Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Example for Variational Autoencoder for Collaborative Filtering for Implicit Feedback Datasets (Citeulike)"""
+
+import cornac
+from cornac.data import Reader
+from cornac.datasets import citeulike
+from cornac.eval_methods import RatioSplit
+
+_, item_ids = citeulike.load_text()
```

Comment:
Loading item_ids is unnecessary in this case.

Reply:
Thank you for your review.

Reply:
Welcome :). Lines 18 (import Reader) and 22 (load_text()) can be removed.

Reply:
Almost there :). Please consider removing line 18: `from cornac.data import Reader`.